swap_meet/vendor.py

```diff
@@ -1,2 +1,93 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if not self.inventory or item not in self.inventory:
+            return False
+        self.inventory.remove(item)
+        return item
+    
+    def get_by_id(self, id):
+        try:
+            return [item for item in self.inventory if item.id == id][0]
```

Comment:
[perf] This is nice and succinct, but you might be better off using a `for` loop since you can early return if the item is found. (This list comprehension has to iterate the whole list to find all `id`s that match `id`.